src/ReleaseApp/Domain/Entities/Collection/UpgradeInstructionModuleCollection.php

```diff
@@ -69,4 +69,34 @@ public function addCollection(self $collectionToMerge): void
     {
         $this->elements = array_merge($this->elements, $collectionToMerge->toArray());
     }
+
+    /**
+     * @param string $name
+     *
+     * @return \ReleaseApp\Domain\Entities\UpgradeInstructionModule|null
+     */
+    public function getByName(string $name): ?UpgradeInstructionModule
+    {
+        foreach ($this->elements as $module) {
```

Comment:
It will be 4 lines, we have to use `array_shift` because I don't like unclear indexes `[0]`

$array = array_filter($this->elements, function (UpgradeInstructionModule $module) use ($name) {
            return $module->getName() === $name;
        });
        return array_shift($array) ?? null;`
`

Reply:
Discussed that it could be lived as it is